[2022-02-27 17:36:16,372][wiiboard][ERROR] average called before any data could get in
[2022-02-27 17:36:18,819][wiiboard][ERROR] average called before any data could get in
[2022-02-27 17:36:22,555][wiiboard][ERROR] average called before any data could get in
[2022-02-27 17:36:24,992][wiiboard][ERROR] average called before any data could get in
[2022-02-27 17:36:27,421][wiiboard][ERROR] average called before any data could get in
[2022-02-27 17:36:29,840][wiiboard][ERROR] average called before any data could get in
[2022-02-27 17:36:32,237][wiiboard][ERROR] average called before any data could get in
[2022-02-27 17:36:34,644][wiiboard][ERROR] average called before any data could get in
[2022-02-27 17:36:37,060][wiiboard][ERROR] average called before any data could get in
[2022-02-27 17:36:39,498][wiiboard][ERROR] average called before any data could get in
Exception in thread Thread-1:
Traceback (most recent call last):
  File "<string>", line 3, in recv
_bluetooth.error: (103, 'Software caused connection abort')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 892, in run
    self._target(*self._args, **self._kwargs)
  File "/mnt/bee1/wiibee/wiiboard.py", line 121, in loop
    data = self.receivesocket.recv(25)
  File "<string>", line 5, in recv
bluetooth.btcommon.BluetoothError: [Errno 103] Software caused connection abort
[2022-02-27 17:39:51,977][wiiboard][ERROR] average called before any data could get in
[2022-02-27 17:39:54,420][wiiboard][ERROR] average called before any data could get in
[2022-02-27 17:39:56,839][wiiboard][ERROR] average called before any data could get in
[2022-02-27 17:39:59,239][wiiboard][ERROR] average called before any data could get in
[2022-02-27 17:40:01,639][wiiboard][ERROR] average called before any data could get in
[2022-02-27 17:40:04,051][wiiboard][ERROR] average called before any data could get in
[2022-02-27 17:40:06,484][wiiboard][ERROR] average called before any data could get in
[2022-02-27 17:40:08,920][wiiboard][ERROR] average called before any data could get in
[2022-02-27 17:40:11,331][wiiboard][ERROR] average called before any data could get in
[2022-02-27 17:40:13,732][wiiboard][ERROR] average called before any data could get in
Exception in thread Thread-1:
Traceback (most recent call last):
  File "<string>", line 3, in recv
_bluetooth.error: (103, 'Software caused connection abort')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 892, in run
    self._target(*self._args, **self._kwargs)
  File "/mnt/bee1/wiibee/wiiboard.py", line 121, in loop
    data = self.receivesocket.recv(25)
  File "<string>", line 5, in recv
bluetooth.btcommon.BluetoothError: [Errno 103] Software caused connection abort
